cogs/misc.py

```diff
@@ -44,6 +47,34 @@ async def _8ball(self, ctx, *, question=None):
             random.choices(tuple(outcomes.keys()), tuple(outcomes.values()))[0]
         )
 
+    @commands.command(name="proton")
+    async def proton(self, ctx):
+        """Retrieve proton stats for Project Winter."""
+
+        url = "https://www.protondb.com/api/v1/reports/summaries/774861.json"
+
+        useful_info = {
+            "confidence": "Confidence",
+            "tier": "Tier",
+            "bestReportedTier": "Best Reported Tier",
+            "total": "Reviews",
+        }
+
+        embed = Embed(
+            title="Project Winter Proton Statistics",
+            colour=Colour.red()
```

Comment:
Can't this change depending on the rating? Do we maybe not have a good set of colours to reflect this?